src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const editWords = (words) => words.trim();
```

Comment:
[CODE STYLE] - The function 'editWords' could be named more descriptively to reflect its purpose, such as 'trimWhitespace'.